rpm/busybox.spec

```diff
@@ -1,6 +1,6 @@
 Summary: Single binary providing simplified versions of system commands
 Name: busybox
-Version: 1.34.1
+Version: 1.36.1
 Release: 1
 License: GPLv2
```

Comment:
Please also fix the license tag.

Reply:
This is currently according our policy so it should be kept as such. We are using the Legacy Fedora Abbreviation notation still.

Reply:
There is nothing wrong with this license tag AFAICT.

Reply:
Our policy is to follow SPDX license tags similar as Fedora does GPLv2 is invalid according to that it should be GPL-2.0.
https://docs.fedoraproject.org/en-US/legal/license-field/

Reply:
Our packaging policy is defined on high level at https://docs.sailfishos.org/Reference/Software_Packaging/#RPM_packaging_Guidelines which refers to fedora policy. Fedora policy has indeed changed since, but even in fedora pages e.g. https://docs.fedoraproject.org/en-US/legal/allowed-licenses/ they have still "Legacy Abreviation" list which is the one that we are still following, we have not done decision to move to SPDX yet on these.